Source/WebKit/UIProcess/API/Cocoa/WKNavigationResponsePrivate.h

```diff
@@ -26,10 +26,13 @@
 #import <WebKit/WKNavigationResponse.h>
 
 @interface WKNavigationResponse (WKPrivate)
-
-@property (nonatomic, readonly) WKFrameInfo *_frame;
+NS_ASSUME_NONNULL_BEGIN
+@property (nonatomic, readonly) WKFrameInfo * _frame;
 @property (nonatomic, readonly) NSURLRequest *_request;
 @property (nonatomic, readonly) NSString *_downloadAttribute WK_API_AVAILABLE(macos(10.15), ios(13.0));
+NS_ASSUME_NONNULL_END
```

Comment:
This typically encompasses the entirety of the file.  Rather than update all the other attributes, let's omit explicitly stating nullable below.